packages/elements-core/src/components/Docs/HttpOperation/Parameters.tsx

```diff
@@ -74,10 +76,11 @@ export const Parameter: React.FunctionComponent<IParameterProps> = ({ parameter,
   const schemaExamples = parameter.schema?.examples;
   const schemaExamplesArray = Array.isArray(schemaExamples) ? schemaExamples : [];
 
-  const validations = omitBy(
+  let validations = omitBy(
```

Comment:
hm, why `const` -> `let` change?

Reply:
No need, a leftover from some experiments. Changing it back 👍